src/importer.cpp

```diff
@@ -343,7 +343,7 @@ void Importer::ImporterImpl::makeIssueCyclicDependency(const ModelPtr &model,
     auto issue = Issue::create();
     issue->setDescription(msg);
     issue->setLevel(Issue::Level::WARNING);
-    issue->setCause(ItemType::IMPORT);
+    issue->setReferenceRule(Issue::ReferenceRule::IMPORT_EQUIVALENT);
     mImporter->addIssue(issue);
```

Comment:
Need to set an item here - probably the import source?

Reply:
There is no import source available at this point.

Reply:
Yes there is, it just needs to be passed in through the arguments of this function.  Otherwise this becomes an orphaned issue, but we do have things that it could be tied to.